lib/views/main_view.dart

```diff
@@ -1,9 +1,30 @@
 import 'package:aimimi/styles/colors.dart';
+import 'package:aimimi/views/leaderboard_view.dart';
+import 'package:aimimi/views/profile_view.dart';
 import 'package:aimimi/views/today_view.dart';
+import 'package:aimimi/views/goal_view.dart';
 import 'package:flutter/material.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 
-class MainView extends StatelessWidget {
+class MainView extends StatefulWidget {
+  @override
+  _MainViewState createState() => _MainViewState();
+}
+
+class _MainViewState extends State<MainView> {
+  int _currentIndex = 0;
+  final List<Widget> _children = [
```

Comment:
Change _children to _views for readability?

```suggestion
  final List<Widget> _views = [
```

Reply:
Ok. Good idea.